poker/utils/message.py

```diff
@@ -0,0 +1,148 @@
+            player_response = input(f"The bet was raised by {raise_amount}, what would you like to do? [{Action.CALL} or {Action.FOLD}] ").lower()
+            if player_response not in [Action.CALL, Action.FOLD]:
+                raise InvalidActionException
+        else:
+            player_response = input(f"What would you like to do? [{Action.CHECK}, {Action.RAISE} or {Action.FOLD}] ").lower()
+            if player_response not in [Action.CHECK, Action.RAISE, Action.FOLD]:
+                raise InvalidActionException
+        return player_response
+
+    def action_taken(self, name: str, action: str, amount: int, possible_actions: list[str] = []) -> None:
+        print(f"{name} decided to {action} {amount if action in possible_actions else ''}") 
+    
+    def increase(self, chip_count: int) -> int:
+        player_response = int(input(f"How much would you like to raise? "))
+        if player_response <= 0:
+            raise NegativeException
+        if player_response >= chip_count:
+            raise InsufficientChipException
+        return player_response
+    
+    def different_amount(self, chip_count: int) -> None:
+        print(f"You only have {chip_count} chips. Please select a lower amount.")
+    
+    def player_summary(self, players: dict) -> None:
+        player_table = Table(title="Player Summary")
+        player_table.add_column("Order")
+        player_table.add_column("Name")
+        player_table.add_column("Chips")
+        player_table.add_column("Blind")
+        player_table.add_column("Pocket Cards")
+        player_table.add_column("Best Hand")
+        player_table.add_column("Short Name")
+        for player_id, player in players.items():
+            player = player["player"]
+            player_table.add_row(
+                str(player_id), 
+                player.name, 
+                f"{GameStack.WHITE['name']}: {player.stack.chips[GameStack.WHITE['name']]}",
+                "Big" if player_id == 1 else "Small",
+                " ".join(f"{card}" for card in player.pocket_cards) if player.name == PLAYER_NAME or len(player.best_hand) > 1 else "Hidden",
+                ", ".join(f"{card}" for card in player.best_hand["hand"]) if len(player.best_hand) > 1 else "",
+                player.best_hand["short"] if len(player.best_hand) > 1 else ""
+                )
+        console = Console()
+        console.print("", player_table)
+    
+    def game_progression_prompt(self, progress: bool = None) -> None:
+        try:
+            if progress:
+                player_response = input(f"How about now, are you ready? [yes/no] ").lower()
+            else:
+                player_response = input(f"Are you ready to continue? [yes/no] ").lower()
+            if player_response not in self.decision:
+                raise GamePlayException
+            if player_response in [Decision.N.value, Decision.NO.value]:
+                raise NotReadyException
+        except GamePlayException:
```

Comment:
I wonder if raise + catch in same place is kind of redundant, can you just do 

```
if player_response not in self.decision:
    # code that happens under "except ..."
```

Reply:
Good call. It is redundant. Your suggestion works much better.